db/structs.go

```diff
@@ -460,14 +461,27 @@ type BudgetStoreData struct {
 	Created      *time.Time `json:"created"`
 }
 
+type BudgetPaymentType string
+
+const (
+	Add     BudgetPaymentType = "add"
+	Deposit BudgetPaymentType = "deposit"
+)
+
 type BudgetHistory struct {
-	ID           uint       `json:"id"`
-	OrgUuid      string     `json:"org_uuid"`
-	Amount       uint       `json:"amount"`
-	SenderPubKey string     `json:"sender_pubkey"`
-	Created      *time.Time `json:"created"`
-	Updated      *time.Time `json:"updated"`
-	Status       bool       `json:"status"`
+	ID           uint              `json:"id"`
+	OrgUuid      string            `json:"org_uuid"`
+	Amount       uint              `json:"amount"`
+	SenderPubKey string            `json:"sender_pubkey"`
+	Created      *time.Time        `json:"created"`
+	Updated      *time.Time        `json:"updated"`
+	Status       bool              `json:"status"`
+	PaymentType  BudgetPaymentType `json:"payment_type"`
+}
+
+type BudgetHistoryData struct {
+	BudgetHistory
+	SenderName string `json:"sender_name"`
```

Comment:
why wouldnt we just fetch from the People table to get the sender name or just include this in BudgetHistory?

Reply:
oh wait I guess this isnt a new table so this should be ok